designsafe/static/scripts/data-depot/components/projects/pipeline-licenses/pipeline-licenses.component.html

```diff
@@ -63,7 +63,7 @@
         <div style="padding-top: 40px; padding-bottom: 40px;">
             <p>
                 <h3 style="text-align: center">Select License(s)</h3>
-                <h3 style="text-align: center; color: orange;">
+                <h3 ng-if="!$ctrl.isOther()" style="text-align: center; color: orange;">
```

Comment:
I know it looks pretty ugly, but for the sake of making it easier to clean up later. Let's just make this expression `$ctrl.project.value.projectType === 'other'` That way when we're trying to find=>replace all these things we can get them in one swoop.